### DISCUSSION TITLE
Add documentation "what curl __cannot__ do" and disclaimer

### DISCUSSION THREAD

**Discussion by ghost** [curl/curl#8501] [Ideas]
1. cURL cannot process any JavaScript or bypass captchas
2. cURL cannot show any fancy CSS styles or colours
3. cURL cannot debug your server code or probe any vulnerabilities
4. cURL is an HTTP  _client_ and does not function as a server
5. cURL is not a VPN or proxy, thus it cannot bypass local censorship.
6. **If you use cURL for malicious purposes and the website/server blocks you, don't blame us, it is completely _your_ responsibility.**
7. [!!] **The curl tool and library itself are open source, but if you want complete support including SLA, you have to pay for it otherwise developers are not obligated to answer any questions.**

[Incomplete and non-exhaustive list]

**Comment by bagder** [comment]
We have a little of that in the FAQ entry [what is curl not](https://curl.se/docs/faq.html#What_is_curl_not). I'm not convinced we need to provide such a list of things **it isn't** but that the focus should be to clearly describe what **it is**.

**Comment by ghost** [comment]
thank you at least include the last two in please

**Reply by jay** [reply]
I think that's unnecessary. It's implied that the user is responsible for how they use the tool. Commercial support is already covered in the FAQ.
